### DISCUSSION TITLE
strapi.components is not loading in bootstrap.js (neither strapi.models)

### DISCUSSION THREAD

**Discussion by admartinez** [strapi/strapi#6374] [Legacy Topics]
strapi.components is not loading in bootstrap.js (neither strapi.models)

**Comment by lauriejim** [comment]
Hello! Ii just tried the following code and i'm able to see models and components.
**Path —** `config/functions/bootstrap.js`
```js
module.exports = () => {
  console.log(Object.keys(strapi.models));
  console.log(Object.keys(strapi.components));
};
```
